apps/accelerate/speedster/docs/en/docs/advanced_options.md

```diff
@@ -198,7 +203,26 @@ input_data = [((torch.randn(1, 3, 256, 256),), torch.tensor([0])) for _ in range
 # Set dynamic info
 dynamic_info = {
     "inputs": [
-        {0: "batch", 2: "dim_image", 3: "dim_image"}
+        {
+            0: {
+                "name": "batch",
+                "min_val": 1,
```

Comment:
is it val or shape? In the code it seems to be shape, but in the docs you put val

Reply:
We use val to indicate the single axis dimension (each input has a shape, where each dimension has a min/max/opt val), while shape is the key used by tensor rt to compile the model with dynamic shape enabled. I called it differently because I didn't like using shape to refer to the single dimension of the shape.